app/views/admin/fact_check_requests/show.html.erb

```diff
@@ -4,6 +4,7 @@
   <div class="col-md-8">
     <section class="document_page" id="<%= dom_id(@edition) %>">
       <%= render partial: 'admin/editions/edition', locals: { edition: @edition } %>
+      <%= render partial: 'admin/editions/show_attachments', locals: { edition: @edition } %>
```

Comment:
I'm not sure what effect this change has. Looking at the screenshots and description in the PR, I can see the tab changes in `edit.html.erb` make sense, but I'm not sure what _this_ change is. Could you clarify?

Reply:
I'll update the PR with a screenshot etc, but also answer here for clarity. On submitting a fact-check the user is presented with this partial with a message thanking them for submitting their review. It also displays the the document so it seemed appropriate to also present the attachments once more.

Reply:
Understood. It doesn't necessarily need a screenshot, but would be good to add that context to the commit message.